src/ui/components/results/paginationcomponent.js

```diff
@@ -67,6 +67,17 @@ export default class PaginationComponent extends Component {
         this.setState();
       }
     });
+
+    /**
+     * Configuration for the behavior when there are no vertical results.
+     */
+    this._noResults = config.noResults || this.core.globalStorage.getState(StorageKeys.NO_RESULTS_CONFIG);
+    if (typeof this._noResults !== 'object') {
```

Comment:
I'm not super familiar with typeof checks in JS, since noResults is optional config, does this need a truthy check to make sure noResults is present before the typeof?

Reply:
Yeah you're right it does, I'll just remove this check I don't think it's particularly useful either way

Reply:
typeof should just return 'null' or 'undefined' in that case